language/move-prover/interpreter/src/concrete/runtime.rs

```diff
@@ -127,9 +127,9 @@ fn check_and_convert_type_args_and_args(
     let mut converted_args = vec![];
     for (i, (arg, param)) in args.iter().zip(params.into_iter()).enumerate() {
         let local_ty = fun_env.get_local_type(i);
-
-        #[cfg(debug_assertions)]
-        assert_eq!(local_ty, param.1);
+        if cfg!(debug_assertions) {
+            assert_eq!(local_ty, param.1);
```

Comment:
Is there `debug_assert_eq`?

Reply:
Yup, updated to `debug_assert_eq`, which is much cleaner.